pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -41,18 +42,23 @@ func (f *frontend) putOrPatchOpenShiftCluster(w http.ResponseWriter, r *http.Req
 	subId := chi.URLParam(r, "subscriptionId")
 	resourceProviderNamespace := chi.URLParam(r, "resourceProviderNamespace")
 
+	identityUrlHeader := "x-ms-identity-url"
+	if r.Header.Get(identityUrlHeader) != "" {
```

Comment:
`Header.Get()` returns the first value associated with the key. Since we expect only one key, we should be good to just pass a string to `_putOrPatchOpenShiftCluster` instead of a slice of strings.